modules/core/arrow-test/src/main/kotlin/arrow/test/laws/SelectiveLaws.kt

```diff
@@ -0,0 +1,44 @@
+package arrow.test.laws
+
+import arrow.Kind
+import arrow.core.*
+import arrow.test.generators.either
+import arrow.typeclasses.Eq
+import arrow.typeclasses.Selective
+import io.kotlintest.properties.Gen
+import io.kotlintest.properties.forAll
+
+object SelectiveLaws {
+
+  fun <F> laws(A: Selective<F>, EQ: Eq<Kind<F, Int>>): List<Law> =
+    ApplicativeLaws.laws(A, EQ) + listOf(
+      Law("Selective Laws: identity") { A.identityLaw(EQ) },
```

Comment:
Did they have more descriptive names on the paper?

Reply:
Distributivity, Associativity, selectM == select if a Monad. 
I can add.